setu/deeplink.py

```diff
@@ -139,3 +139,25 @@ def trigger_mock_payment(self, amountValue, upiID):
         )
 
         return response
+    
+    # Set Bill expiry UPI payment link 
+    def set_bill_expiry(
```

Comment:
Please rename this to a more accurate verb like `expire_bill` since this is the exact function users of the SDK will be calling.